pyproject.toml

```diff
@@ -27,7 +27,7 @@ dependencies = [
     "numpy >=1.21.0",
     "h5py>=3.8.0",
     "tqdm>=4.65.0",
-    "typing_extensions==4.4.0",
+    "typing_extensions>=4.4.0",
```

Comment:
Hi, I changed the title.

I don't think we should remove the dependency, since we still need `Annotated`, this is not support for lower python version like 3.8